src/app/common/utils/spam-filter.ts

```diff
@@ -0,0 +1,23 @@
+const urlRegex =/(((http|ftp|https):\/\/)?(((http|ftp|https):\/\/)?(([\w.-]*)\.([\w]*))))/g;
+const spamWords = ["won", "win", "click"];
+export const spamReplacement = 'Unknown token';
+
+function spamUrlFilter(input: string): boolean {
+    return input.match(urlRegex) ? true : false;
```

Comment:
expression is already a boolean

```suggestion
    return input.match(urlRegex);
```

Reply:
Maybe I shouldn't have a `boolean` return type then? 

If I don't do this ternary TS complains:
![Screenshot 2023-08-09 at 12 00 44](https://github.com/hirosystems/wallet/assets/2938440/58d5c043-0faf-44c0-b502-9ce3773dc439)

Reply:
OK, never-mind. Not defining a return type lets it be inferred properly 👍

Reply:
(Your screenshot was missing a `return` statement)